packages/core/src/App/Containers/AccountSwitcherWallet/account-switcher-wallet.tsx

```diff
@@ -53,15 +53,19 @@ export const AccountSwitcherWallet = observer(({ is_visible, toggle }: TAccountS
             <ThemedScrollbars height={450}>
                 <AccountSwitcherWalletList wallets={dtrade_account_wallets} closeAccountsDialog={closeAccountsDialog} />
             </ThemedScrollbars>
-            <div className='account-switcher-wallet__looking-for-cfds'>
+            <div
+                className='account-switcher-wallet__looking-for-cfds'
+                onClick={handleTradersHubRedirect}
+                onKeyDown={handleTradersHubRedirect}
+                role='button'
+            >
```

Comment:
mmm @lubega-deriv 
this role="button" generate more soundcloud errors
in my PR's i just replaced `div` tag with `button` tag
```suggestion
            <button
                className='account-switcher-wallet__looking-for-cfds'
                onClick={handleTradersHubRedirect}
                type='button'
            >
```

Reply:
![Screenshot 2024-03-21 at 10 04 24](https://github.com/binary-com/deriv-app/assets/103181646/cb6ef7f9-0fd3-4b39-84bb-b835a60212f6)

Reply:
Updated